$MirOS: ports/security/libgcrypt/patches/patch-configure_ac,v 1.2 2008/04/03 21:28:13 tg Exp $
$OpenBSD: patch-configure,v 1.3 2004/05/08 06:41:47 sturm Exp $
--- configure.ac.orig	Fri Apr 25 15:52:29 2008
+++ configure.ac	Tue Aug 26 18:55:51 2008
@@ -29,8 +29,6 @@ min_automake_version="1.10"
 m4_define([my_version], [1.4.1])
 m4_define([my_issvn], [no])
 
-m4_define([svn_revision], m4_esyscmd([echo -n $( (svn info 2>/dev/null \
-            || echo 'Revision: 0')|sed -n '/^Revision:/ {s/[^0-9]//gp;q;}')]))
 AC_INIT([libgcrypt], 
         [my_version[]m4_if(my_issvn,[yes],[-svn[]svn_revision])],
         [bug-libgcrypt@gnupg.org])
@@ -253,9 +251,9 @@ esac
 # Figure out the name of the random device
 #
 case "${host}" in
-    *-openbsd*)
+    *-openbsd* | *-ekkobsd* | *-mirbsd*)
         NAME_OF_DEV_RANDOM="/dev/srandom"
-        NAME_OF_DEV_URANDOM="/dev/urandom"
+        NAME_OF_DEV_URANDOM="/dev/arandom"
         ;;
 
     *)
